MyCryptoHQ/MyCrypto
# Added 0x Instant Swap Fee for MyCrypto

### Description

Adds 3% fee for MyCrypto 0x swaps to help generate revenue.

### Changes

* Added constant for fee receiver address.
* Added constant for fee percentage (as a decimal). (set at 3% - 0.03)

### Steps to Test

1. Open the swap tab at https://mycrypto.com/swap/0x
2. Initiate a swap to buy 5 Dai.
3. Notice that there is now a fee of 3%.
4. Send the swap.
5. Once it completes, navigate to the fee receiver address and notice that it has received ETH in the internal transactions section of Etherscan.

### Screenshots

![0x fee 1](https://user-images.githubusercontent.com/29407814/50980358-82ad8200-14c6-11e9-8e31-bebd64573357.PNG)


![0x fee 2](https://user-images.githubusercontent.com/29407814/50980363-850fdc00-14c6-11e9-8165-c1460c07aeb1.PNG)


![0x fee 3](https://user-images.githubusercontent.com/29407814/50980369-87723600-14c6-11e9-8354-7655c1d4ad0c.PNG)
